izadji/src/pages/DocumentationPage/DocumentationPage.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactNode, useEffect, useState } from 'react';
+
+import 'swiper/scss';
+import 'swiper/scss/pagination';
+import documentationPageService from '../../services/documentationService';
+import { MapComponents } from '../../utils/mappers/sharedMapper';
+import Publications from '../../components/Publications/Publications';
+import { PublicationProps } from '../../components/Publication/Publication';
+
+const GetPublications = (publication: any): PublicationProps => {
```

Comment:
Bit weird to use publicationProps for documentation, even if they are the same.
We should either dupli ate the props, so they change separatly, or extract them (and rename them) so we can reuse them.

Reply:
Agreed, extracted it to DownloadableProps